### PR TITLE
GEODE-6908: retried REMOVE should not create new version tag.

### PR DESCRIPTION
**Pull request by gesterzhou**
added retry dunit tests for all the c/s operations.

Thank you for submitting a contribution to Apache Geode.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ ] Is there a JIRA ticket associated with this PR? Is it referenced in the commit message?

- [ ] Has your PR been rebased against the latest commit within the target branch (typically `develop`)?

- [ ] Is your initial contribution a single, squashed commit?

- [ ] Does `gradlew build` run cleanly?

- [ ] Have you written or updated unit tests to verify your changes?

- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?

### Note:
Please ensure that once the PR is submitted, check Concourse for build issues and
submit an update to your PR as soon as possible. If you need help, please send an
email to dev@geode.apache.org.

### CODE REVIEW

File: geode-core/src/main/java/org/apache/geode/internal/cache/LocalRegion.java [hunk 1]
```diff
@@ -8429,6 +8432,8 @@ void clearRegionLocally(RegionEventImpl regionEvent, boolean cacheWrite,
     }
 
     RegionVersionVector myVector = getVersionVector();
+    System.out.println(Thread.currentThread().getName() + ": LocalRegion.clearRegionLocally region="
```
Comment (bschuchardt) [inline geode-core/src/main/java/org/apache/geode/internal/cache/LocalRegion.java]: you've got a debug println that needs to be removed

Comment (gesterzhou) [reply geode-core/src/main/java/org/apache/geode/internal/cache/LocalRegion.java]: Oh, it's from another on-going bug fix. I removed it.

File: geode-core/src/main/java/org/apache/geode/internal/cache/LocalRegion.java [hunk 2]
```diff
@@ -3101,7 +3101,7 @@ boolean cacheWriteBeforeRegionDestroy(RegionEventImpl event)
     return result;
   }
 
-  private void cacheWriteBeforeRegionClear(RegionEventImpl event)
+  protected void cacheWriteBeforeRegionClear(RegionEventImpl event)
```
Comment (pivotal-eshu) [inline geode-core/src/main/java/org/apache/geode/internal/cache/LocalRegion.java]: Do you really need to increase the visibility here?

Comment (gesterzhou) [reply geode-core/src/main/java/org/apache/geode/internal/cache/LocalRegion.java]: fixed

**Pull request merged**